src/Tizen.NUI/src/public/Visuals/VisualObject/VisualBase.cs

```diff
@@ -358,20 +366,32 @@ public float Opacity
         /// The default value is VisualFittingModeType.DontCare.
         /// If user set one of Transform property, it will be set as VisualFittingModeType.DontCare automatically.
         /// </remarks>
+        /// <remarks>
+        /// Fitting mode is only avaliable when the visual has original size.
+        /// For example, ImageVisual and TextVisual support FittingMode, but ColorVisual and BorderVisual doesn't support.
+        /// If visual doesn't have original size, Property set will be ignored.
+        /// </remarks>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public VisualFittingModeType FittingMode
         {
             set
             {
-                if (value != VisualFittingModeType.DontCare)
+                if (IsFittigModeAvaliable())
```

Comment:
Fix typo : (Same as the original one below)

```suggestion
                if (IsFittingModeAvailable())
```

Reply:
Oh! my mistack :( Fix done.